.github/workflows/integ_test_datastore_v2.yml

```
@@ -0,0 +1,59 @@
+name: DataStore Integration V2 Tests
```

I think referencing TransformerV2 in the name, which shows up as the name of the workflow under Actions tab, will make more clear that we're talking about TransformerV2 and not some other version like the library V2 version
```suggestion
name: DataStore TransformerV2 Tests
```